typescript.js

```diff
@@ -32,7 +32,19 @@ module.exports = {
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-non-null-assertion": "error",
         "@typescript-eslint/no-require-imports": "error",
-        "@typescript-eslint/no-unused-vars": ["error", { args: "after-used", argsIgnorePattern: "[iI]gnored$" }]
+        "@typescript-eslint/no-unused-vars": ["error", { args: "after-used", argsIgnorePattern: "[iI]gnored$" }],
+        "@typescript-eslint/ban-types": ["error", {
```

Comment:
you can just do

`"@typescript-eslint/ban-types": "error",`

built-int types are banned per default, have a look here: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/rules/ban-types.ts#L83

Reply:
Also, I think this list is still sorted alphabetically, so please move this rule up ;)

Reply:
covered in [e463d2e8](https://github.com/pmedianetwork/eslint-config/pull/44/commits/e463d2e85af6ccf87f8c51896ad3a4667bff24f1)